### DISCUSSION TITLE
Appeal for Tasks/Points

### DISCUSSION THREAD

**Discussion by taramakage** [game-of-nfts/gon-evidence#921] [General]
If you have any doubts about your task points, please appeal in the discussion:

- Please **provide the name of the task** for which you believe there is a problem. We will re-examine the task and inform you of the reason for any point deductions.
- During the batch verification process, some transaction captures may be lost due to network fluctuations. For these tasks, we will replenish any missing points after re-examination.
- If points were deducted due to **missing evidence, incorrect evidence, or formatting errors**, we will not adjust your score. We have previously emphasized [the importance of proper formatting](https://github.com/game-of-nfts/gon-evidence/issues/376).
- Please note that we are no longer accepting PRs related to evidence. 
- The appeal period will end on **April 4th at 24:00 UTC**.

Please use the following **template** when submitting your appeal in this discussion:

```
- Task: A5, A6, A10
- Link: Link to your evidence file in gon-evidence.
```
**Please do not occupy too many floors in this discussion. One floor per person is enough.**

Note some Juno related tasks have been fixed (due to a previous endpoint issue). So please check the newest socrecard before submitting the appeal. You can also use the `gon-verifier` tool to see details why you lost points. 

**Race Ranking** related scores will be subject to point correction.

**Comment by Dedenwrg** [comment]
- Task: A15,  A16, A17, A19, B1, B2, B5, B6, B7
- Link: https://github.com/game-of-nfts/gon-evidence/blob/main/deden2904/evidence.xlsx

**Reply by taramakage** [reply]
A15 to A19, B1 to B6 Fixed. B7 has a wrong sheet name so it won't be calculated.

**Comment by secard51** [comment]
Link > https://github.com/game-of-nfts/gon-evidence/blob/main/secard51/evidence.xlsx

Task > A3,A4,A5,A6,A15,A16,A17,A18,A19,B5

**Reply by taramakage** [reply]
B5: unexpected race flow
A3, A5, A6 all have wrong evidence
A18: initial owner not match register address.
Others are fixed.

**Reply by secard51** [reply]
I made the first update for a3 and a5, it was approved after your warning, if you look at the history of excel, you can confirm me, I opened a new evidence pr

**Reply by secard51** [reply]
my evidence files are messed up

**Comment by farukyasar** [comment]
Link >https://github.com/game-of-nfts/gon-evidence/blob/main/farukyasar/evidence.xlsx
Tasks: A5, A9, A15 A16, A17, A19, A20, B1 ,B2

**Reply by taramakage** [reply]
B2 unexpected flow.
A9 original class id not match
A16 IBC: dest port not match
A19 Tx: sender not match register address
A20 Tx: result is unachievable from stargaze

Others are fixed.

**Reply by farukyasar** [reply]
B2 > ✓
A9 > ✓
A16 > ✓
A20 > https://gon.ping.pub/stargaze/tx/9534E2CF897A6383A5451265644823C3C63759D5F7EBAA94C4554A135D0CAEFF
A19 > https://gon.ping.pub/iris/tx/0BF73F00278571FF93595B33CD4D16DF828E646CA40F9FCEE9B6E2049538B166
          Sender **iaa1pa08ws4mhvp2qcpz5yh4yfqu0rj8hzxwv8fpgz** my evidence IRIS address

**Reply by taramakage** [reply]
A20: will you note this failed tx 9B4A0C7CEE82A79D3C30C9128C45C0D3F54D3E742AB0925CEE7E6F2267FF25EB ?
A19: it's that one of your addr in tx(from iris to omniflix) didn't not match register addr.

**Reply by farukyasar** [reply]
is there any time for update? just TX update. @taramakage 
i found the correct one 
B414175F24A9F3E98C28B99EDCEDED0F6600AB9C486DC515EDA3FDEA6870CEB3

**Reply by taramakage** [reply]
>  If points were deducted due to missing evidence, incorrect evidence, or formatting errors, we will not adjust your score.

Sorry, please take note of this.

**Reply by farukyasar** [reply]
> Comment options

iris to omni? or iris to stars?
https://gon.ping.pub/stargaze/tx/E079BA594D88FE6F24627B783B9A159DD52926197D47C313FEE199FF01014063

**Comment by IAmScRay** [comment]
Task: A20
Link: https://github.com/game-of-nfts/gon-evidence/blob/main/IAmScRay/evidence.xlsx

I saw that there's no acknowledgement packet for TX on Omniflix when doing transfer from Omniflix to Stargaze (TxID: 0816E501A384AC9A5BB1B5379432E62FD3A1E5B1B7D7AC4DEEFE1CF993B99F63) despite having successful transfer and getting my NFT back to Stargaze after all.

**Reply by taramakage** [reply]
A20: Tx result is unachievable

**Reply by IAmScRay** [reply]
> A20: Tx result is unachievable

I don't get it. Was the task completed or not? And which TX result is not achievable?

@taramakage

**Comment by arifintahu** [comment]
Link > https://github.com/game-of-nfts/gon-evidence/blob/main/arifintahu/evidence.xlsx
Tasks > A15, A16, A17, A19

**Reply by taramakage** [reply]
fixed

**Comment by silentnoname** [comment]
Link >  https://github.com/game-of-nfts/gon-evidence/blob/main/silentnoname/evidence.xlsx
Task > A15, A16, A17, A19

**Reply by taramakage** [reply]
A15, A16, A17, A19 NFT: recipient not match register address

**Reply by silentnoname** [reply]
Thank you, Just found that I inputed my star address in the juno address field 😢

**Comment by DreamGallery** [comment]
Task: A2, A5, A6, B7
Link: https://github.com/game-of-nfts/gon-evidence/blob/main/DreamGallery/evidence.xlsx

**Reply by taramakage** [reply]
A2: nft data is empty
A5, A6: chainId is error

**Comment by kaustubhkapatral** [comment]
Task: A7, A8, A9, A10, A11, A12
Link: https://github.com/game-of-nfts/gon-evidence/blob/main/kaustubhkapatral/evidence.xlsx

**Reply by taramakage** [reply]
All have the same error: IBC: original class id not match

**Reply by kaustubhkapatral** [reply]
@taramakage Would it be possible to explain it a bit more? Because the same class id was used for all tasks and the flow ids were for these specific tasks were followed properly.

**Reply by taramakage** [reply]
All your input in these tasks are original class ids, not ibc/class ids. :disappointed:

**Reply by kaustubhkapatral** [reply]
Well shit. Guess I misread it.

**Comment by nodeiistt** [comment]
Task: A4, A6, A8, A15, A16, A17
Link: https://github.com/game-of-nfts/gon-evidence/blob/main/nodeiistt/evidence.xlsx

**Reply by taramakage** [reply]
A4: Tx: result is unachievable
A6: NFT: recipient not match register address
A8: NFT not found
A15&A16: IBC: dest channel not match
A17 is fixed

**Reply by nodeiistt** [reply]
and 19

**Reply by taramakage** [reply]
A19 fixed

**Comment by omercan12345** [comment]
Task: A15, A16, A17
Link: https://github.com/game-of-nfts/gon-evidence/blob/main/gokceeee/evidence.xlsx

i fixed this tasks please check again: https://github.com/game-of-nfts/gon-evidence/pull/553

**Reply by taramakage** [reply]
Fixed

**Comment by omercanyenigun** [comment]
Task: I fixed and reinstalled everything except B1.

Link: https://github.com/game-of-nfts/gon-evidence/blob/main/omercanyenigun/evidence.xlsx

i fixed this tasks please check again: https://github.com/game-of-nfts/gon-evidence/pull/541

**Reply by taramakage** [reply]
A7: IBC: ibc class not match
A15 A16: IBC: dest channel not match

**Reply by omercanyenigun** [reply]
@taramakage A17 A19?

**Reply by taramakage** [reply]
Others are fixed.

**Comment by Yeksin** [comment]
- Task:  A15, A16, A17, A19
- Link: https://github.com/game-of-nfts/gon-evidence/blob/main/yekssin/evidence.xlsx

**Reply by taramakage** [reply]
Fixed

**Comment by 0xtekgrinder** [comment]
Link > https://github.com/game-of-nfts/gon-evidence/blob/main/0xtekgrinder/evidence.xlsx

Task > A15,A16,A17,A19

**Reply by taramakage** [reply]
Fixed

**Comment by leonikaf** [comment]
Tasks: A1-A8
Link: https://github.com/game-of-nfts/gon-evidence/blob/main/leonikaf/evidence.xlsx

**Reply by taramakage** [reply]
Almost each task have problem. You can install gon-verifier to find details.https://github.com/game-of-nfts/gon-verifier

**Comment by ptts55** [comment]
- Task: A15, A16, A17, A19
- Link: https://github.com/game-of-nfts/gon-evidence/blob/main/ptts55/evidence.xlsx

**Reply by taramakage** [reply]
Fixed

**Comment by williamchong** [comment]
- Task: A15, A16, A17, A19
- Link: https://github.com/game-of-nfts/gon-evidence/blob/main/williamchong/evidence.xlsx

**Reply by taramakage** [reply]
Fixed

**Comment by beal2912** [comment]
- Task: A15, A16, A17, A19
- Link: [https://github.com/game-of-nfts/gon-evidence/blob/main/beal2912/evidence.xlsx](https://github.com/game-of-nfts/gon-evidence/blob/main/beal2912/evidence.xlsx)

the tasks were ok in the score board before the last update so I didn't edit the format, and I used the exact same format on A18 and A20 which are counted as ok

**Reply by taramakage** [reply]
Fixed

**Comment by osmanbo** [comment]
- Task: A5, A15, A16, A17, A19
- Link: [Link to your evidence file in gon-evidence.](https://github.com/game-of-nfts/gon-evidence/blob/main/osmanbo/evidence.xlsx)

**Reply by taramakage** [reply]
A16 dest port not match. Others are fixed.

**Comment by N1ghtSe7en** [comment]
Task: A5, A15, A16, A17, A19
Link: https://github.com/game-of-nfts/gon-evidence/blob/main/N1ghtSe7en/evidence.xlsx

**Reply by taramakage** [reply]
Fixed

**Comment by gruberx** [comment]
- Task: A5, A15, A16, A17, A19
- Link: https://github.com/game-of-nfts/gon-evidence/blob/main/gruberx/evidence.xlsx

**Reply by taramakage** [reply]
Fixed

**Comment by anurg** [comment]
- Task: A12, A14, A15, A16, A17, A19, B1
- Link: https://github.com/game-of-nfts/gon-evidence/blob/main/anurg/evidence.xlsx

**Reply by taramakage** [reply]
A12 A14: NFT not found
B1: unexpected race flow

**Reply by taramakage** [reply]
Others are fixed.

**Comment by whoistiger** [comment]
Task: A1,A2,A3,A4,A5,A6,A9,A11,A12,A13,A14,A15,A16
Link: https://github.com/game-of-nfts/gon-evidence/blob/main/Blockpower/evidence.xlsx

**Reply by taramakage** [reply]
Almost every evidence of these tasks have some problems. You can use gon-verifier to see details.

**Comment by heejin-github** [comment]
Task : A7,A8,A9,A10,A11,A12,A14,A15,A16,A17,A19
Link : https://github.com/game-of-nfts/gon-evidence/blob/main/heejin-github/evidence.xlsx

**Reply by taramakage** [reply]
A7~A11: IBC: original class id not match
A14: NFT: initial owner not match register address
A19: Tx: result is unachievable

**Reply by taramakage** [reply]
A15, A16, A17 are fixed.

**Comment by HashKeyCloud** [comment]
Task：A5
Link：[evidence.xlsx](https://github.com/HashQuark-Research1/gon-evidence-cosmos/tree/main/HashQuark-Research1/evidence.xlsx)

**Reply by taramakage** [reply]
FIXED

**Comment by ciprianmuja** [comment]
Task: A5, A15, A16, A17, A19, B8(top 10)
Link: [evidence.xlsx](https://github.com/game-of-nfts/gon-evidence/blob/main/ciprianmuja/evidence.xlsx)

**Reply by taramakage** [reply]
B8 you ranked 34. Others are fixed.

**Comment by PippinAintEasy-B** [comment]
Our pull request, with our evidence from monday is still open. Can that be resolved so we cant get our score right?
[https://github.com/game-of-nfts/gon-evidence/pull/882](https://github.com/game-of-nfts/gon-evidence/pull/882)

**Reply by taramakage** [reply]
Merged.

**Comment by 2019jack** [comment]
Task: A5, A6, A20
Link: [evidence.xlsx](https://github.com/2019jack/gon-evidence/blob/main/2019jack/evidence.xlsx)

**Reply by taramakage** [reply]
A6: NFT: recipient not match register address

A5 is fixed.

**Reply by 2019jack** [reply]
> A6: NFT: recipient not match register address
> 
> A5 is fixed.
A6 can you check?
https://gon.ping.pub/omniflix/tx/94E4BEE03CD46A473D46F24173C24183DFB11C8B5D31EAD21E2EF40712647148
since omniflixhub transfer IRIS
![o-i](https://user-images.githubusercontent.com/49616005/228770421-20d15c10-e9c1-4a35-a489-6dad5d291773.jpg)

**Reply by 2019jack** [reply]
@taramakage can you again check my task A6, A20, thanks.

**Reply by 2019jack** [reply]
> Task: A5, A6, A20 Link: [evidence.xlsx](https://github.com/2019jack/gon-evidence/blob/main/2019jack/evidence.xlsx)

@taramakage , can you check my task A6, A20?
I have used the gon-verifier tool to verify that my tasks are completed, and there are no problems. Please see the results of my execution
2023/03/31 10:38:13 INFO start to verify TeamName=2019jack " Github"=2019jack
2023/03/31 10:38:16 INFO start to verify TeamName=2019jack " Github"=2019jack
2023/03/31 10:38:18 INFO start to verify TeamName=2019jack " Github"=2019jack
2023/03/31 10:38:19 INFO start to verify TeamName=2019jack " Github"=2019jack
![iris-1](https://user-images.githubusercontent.com/49616005/229009395-9f1f4343-52fc-4a01-a9cc-6d2b13f377f1.jpg)

**Comment by taitruong** [comment]
Same for me: A19 is missing
https://github.com/game-of-nfts/gon-evidence/tree/main/taitruong

**Reply by taramakage** [reply]
Fixed.

**Comment by taitruong** [comment]
And again about the quiz: looking at the scorecard, I figured 11 participants answered these number of questions:

41+6+20+3+15+4+4+3+1+1+1 = 99 total answered questions = 990 points for answered quizzies!

Where highest participant got 410 points and lowest one 10 points. There has been 26 challenges before, we each could achieve - what 67 points? And for addition 50 + 50 + 150 extrapoints in for best 10 participants by solving 5 tasks (B1, B2, B5, B6, B7).
So in total 267 points in 25 tasks. And last task was just about interchain quiz, nothing about interchain transfers and tests. 1000 points for 100 simple quizzies compared to 267 max points for remaining 25 tasks!

I have asked multiple times and got no answers yet, why vote has been outruled, by reasoning this:

"While we understand that many voted to cancel the round in the discussion, we also recognize the possibility that the voting process might have been influenced by the uneven distribution of points among the teams."

Uhm, yes, might be. But is this a reason the outrule voting of majority? Wasn't there a Stargaze proposal 128, where Stargaze did not even supported it, resp saw negative effects for developers, and accepted it got passed because of the majority? Isn't this is what this is all about - community driven?

Organizers put a vote and admitting, assumption was wrong: https://github.com/game-of-nfts/gon-evidence/discussions/671

"We estimated that these tasks would be completed equally among the participants. However, unexpectedly, a large number of NFTs were concentrated in a few accounts, which has compromised the fairness of the game."

After one day of voting it has been decided by organizers, to keep as-is and outruled vote:

![image](https://user-images.githubusercontent.com/776986/228640724-6ff99ef8-82d7-492c-8390-ec99d64e7c2b.png)

Who has decided this? Where all organizers been involved into this?
@jakehartnell @shanev @taramakage @eviefushigi @uptSmart

**Reply by gjermundgaraba** [reply]
I think it was answered in the original post: "While we understand that many voted to cancel the round in the discussion, we also recognize the possibility that the voting process might have been influenced by the uneven distribution of points among the teams."

You can't have a vote to cancel something where most people have strong incentives to vote in a specific way. So that is why they ignored the result, I believe.

And the rules were followed, the game was played, and changing that after-the-fact is not something that should be done. Yeah, the results were not what was planned for, but it was a competitive game, so you can't blame participants for being competitive, either? For myself, I was competitive and put a lot of effort into this one because of the points. If the points were lower I might have done otherwise, but canceling or changing the points now would be a rug pull for those that put a lot of effort and got rewarded for it.

**Comment by taitruong** [comment]
**I'm aware scores won't change anymore, but next time be fair, accept communities decision and have all organizers involved into this!**

**Comment by yamat3h** [comment]
Task: A2
Link: https://github.com/game-of-nfts/gon-evidence/blob/main/yamat3h/evidence.xlsx

**Reply by taramakage** [reply]
A2: NFT: uri is empty

**Comment by alkadeta** [comment]
-Task: A19, B1, B2, B5
-Link: https://github.com/game-of-nfts/gon-evidence/blob/main/alkadeta/evidence.xlsx

**Reply by taramakage** [reply]
B1: NFT: initial owner not match register address
A19 & B2: fixed

**Reply by alkadeta** [reply]
Yeah, I've sent it to the creator instead of the last owner. Could you give like 4 points instead of 0 for that little mistake hahah. Thank you for fixing the two tasks.

What about B5, first tx I shared is the wrong one can I find the real tx and fix?

**Reply by taitruong** [reply]
It would have been even better measuring it, without internal transfers. There is an easy way for finding counterpart TX of transfer on IRISnet side. In terms of metrics this is the most interesting part, figuring out, how fast or slow an interchain transfer is!

Besides, I would give every participant some points due to this little user mistake :).

**Reply by alkadeta** [reply]
Thanks mr T, the task is done in 13 blocks which is I think one of the best timings but due to a little confusion I sent it to the creator wallet lol. I think I at least deserve more than 0 lol

**Comment by kivancbeser** [comment]
-Task: A15, A16, A19,A20, B1,B2
-Link: https://github.com/game-of-nfts/gon-evidence/blob/main/kivancbeser/evidence.xlsx

**Reply by taramakage** [reply]
B1 fixed. B2 unexpected flow.

A15, A16, A19, A20 dest chan not match

**Comment by arnettee** [comment]
-Task: A14, B1,B2
-Link: https://github.com/game-of-nfts/gon-evidence/blob/main/arnettee/evidence.xlsx

**Reply by taramakage** [reply]
B1: fixed.
B2: Race: race flow unexpected
A14:  NFT not found

**Reply by arnettee** [reply]
Thanks for effort. Can you check again my A14:
nftid: nftid14
This is my mint nft tx:
https://gon.ping.pub/iris/tx/EA1784F4A7B00EEA3057D7A1EB283B1CC51957DE61B4AB244F8C899047218560
That is my txhash of last IBC transfer(uptick-iris) A14. 
https://gon.ping.pub/uptick-network-testnet/tx/5F16CF0F8C4B1EC23AFD5BEB2382BC21760D5D300CAF258EA263FBF3B90D256F

**Comment by nurtagum** [comment]
- Task: B2, B5
- Link: https://github.com/game-of-nfts/gon-evidence/blob/main/nurtagum/evidence.xlsx

**Reply by taramakage** [reply]
Fixed.

**Comment by mahofmahof** [comment]
Task: A13
Link:  https://github.com/game-of-nfts/gon-evidence/blob/main/mahofmahof/evidence.xlsx

**Reply by taramakage** [reply]
Format error in A13

**Reply by mahofmahof** [reply]
can i fix it now

**Comment by taitruong** [comment]
What happenedd with B4 I got points from? It is now removed and we lost 50 points. Could you pls provide top 10 for B1 and B2, so we can validate this?


https://github.com/game-of-nfts/gon-evidence/tree/main/taitruong

**Comment by timveltman3** [comment]
- Task: A8
- Link: https://github.com/game-of-nfts/gon-evidence/blob/main/timveltman3/evidence.xlsx

**Comment by ibrahimdulger** [comment]
-Tasks: B5, B6, B7
-Link: https://github.com/ibrahimdulger/gon-evidence/blob/main/ibrahimdulger/evidence.xlsx

**Comment by sarsersirsur** [comment]
- Task: B1, B2, B5
- Link: https://github.com/game-of-nfts/gon-evidence/blob/main/sarsersirsur/evidence.xlsx

**Comment by onurhizlan** [comment]
- Task: B1, B6, B7
- Link: https://github.com/game-of-nfts/gon-evidence/blob/main/onurhizlan/evidence.xlsx

**Comment by bunnygangsta** [comment]
Tasks: A11, A13, A14, A15, A16, B1
Link: https://github.com/game-of-nfts/gon-evidence/blob/main/bunnygangsta/evidence.xlsx

**Comment by gigadudest** [comment]
Tasks: A1, A2, A3, A4, A5, A6, A7, A8, A9, A10, A11, A12, A13, A14, A15, A16, A17, A18, A19, A20
Link: https://github.com/gigadudest/gon-evidence/blob/main/gigadudest/evidence.xlsx

My pr https://github.com/game-of-nfts/gon-evidence/pull/779 merged before deadline. But has wrong folder issue. 
I sent a new pr for correction. https://github.com/game-of-nfts/gon-evidence/pull/947
